trillian_map_api.proto

```diff
@@ -58,7 +59,8 @@ message GetMapLeavesResponse {
 message SetMapLeavesRequest {
   int64 map_id = 1;
   repeated MapLeaf leaves = 2;
-  MapperMetadata mapper_data = 3;
+  reserved 3;
```

Comment:
Maybe `reserved 3;  // MapperMetadata mapper_data (removed)` ?

Reply:
Will do.

Looking back at this I'm bothered by having 'mapper_metadata' in SetMapLeavesRequest but 'metadata' in SignedMapHead.

I'd prefer them to be the same, and think on balance 'metadata' is fine. wdyt?

Reply:
I think metadata is OK. I'd definitely encourage using regular names, it makes for more intuitive APIs.